modules/i18n/src/translations/en-US/portals/console.ts

```diff
@@ -730,8 +731,18 @@ export const console: ConsoleNS = {
                                             }
                                         }
                                     },
-                                    mappedAtributeHint: "Instead of sending the default attribute, the custom " +
-                                        "mapped attribute will be sent",
+                                    mappedAtributeHint: "Enter the custom attribute that should be requested " + 
+                                        "instead of the default attribute.",
+                                    mappingRevert: {
+                                        confirmPrimaryAction: "Confirm",
+                                        confirmSecondaryAction: "Cancel",
```

Comment:
In other confirmation modals, apart from the confirm button, we also have an input field saying something like "Type **Revert attributes** to confirm".

Reply:
In the first phase we'll only use the button confirmations.